config/quickstart-hdfs.properties

```diff
@@ -13,7 +13,6 @@
 name=hdfs-sink
 connector.class=io.confluent.connect.hdfs.HdfsSinkConnector
 tasks.max=1
-topics=test
+topics=test_hdfs
 hdfs.url=hdfs://localhost:9000
-hdfs.conf.dir=/etc/hadoop
```

Comment:
Btw, I think this should have been `hadoop.conf.dir` I think (though it shouldn't matter now because the PR removes this setting altogether).

See src/main/java/io/confluent/connect/hdfs/HdfsSinkConnectorConfig.java.